projects/common/src/utilities/formatters/string/string-formatter.ts

```diff
@@ -1,14 +1,15 @@
-export const titleCaseFromKebabCase = (kebabCaseString: string): string =>
-  kebabCaseString
-    .split('-')
+const titleCaseFromString = (text: string, splitter: string): string =>
+  text
+    .split(splitter)
     .map(str => (str.length === 0 ? str : str[0].toUpperCase() + str.slice(1)))
     .join(' ');
 
-export const titleCaseFromSnakeCase = (snakeCaseString: string): string =>
-  snakeCaseString
-    .split('_')
-    .map(str => (str.length === 0 ? str : str[0].toUpperCase() + str.slice(1)))
-    .join(' ');
+export const titleCaseFromKebabCase = (kebabCaseString: string): string => titleCaseFromString(kebabCaseString, '-');
+
+export const titleCaseFromWhiteSpacedString = (whiteSpacedString: string): string =>
```

Comment:
Is this method similar to Lodash's 'startCase' ?

Reply:
Yep

Reply:
Do we still need this new method? 🙂

Reply:
startCase works for the other cases for which we've defined util methods as well (kebab/snake case to title case). Going by that, my understanding was that if its light enough, we'll define our own?

Reply:
> Going by that, my understanding was that if its light enough, we'll define our own?

I think that goes more for components (as the styling on these evolves) or for util function that we'd need to bring in a new dependency for (such as the whole iso duration thing). If we had no lodash dependency, we shouldn't bring it in for this - but since we do have it and it's already in widespread use, we should be ok to use it where it does exactly what we want.

They might have been written without knowing about start case - or maybe they behave a little differently? If the behavior is identical, then we need not implement it ourselves - although we likely leave the existing ones here for backwards compatibility (could deprecate them, though)

Reply:
I would vote for deprecating them if they are exactly the same.

Reply:
I'd also prefer deprecating them then. startCase works for whitespaced, kebab case and snake cased strings.

Reply:
Deprecated.